ReactNativeClient/lib/services/searchengine/SearchEngine.js

```diff
@@ -235,25 +237,24 @@ class SearchEngine {
 		return occurenceCount / spread;
 	}
 
-	processResults_(rows, parsedQuery) {
+	processResults_(rows) {
 		for (let i = 0; i < rows.length; i++) {
 			const row = rows[i];
-			const offsets = row.offsets.split(' ').map(o => Number(o));
-			row.weight = this.calculateWeight_(offsets, parsedQuery.termCount);
-			row.fields = this.fieldNamesFromOffsets_(offsets);
+			row.weight = 0;
+			row.fields = ['body', 'title'];
 		}
 
-		rows.sort((a, b) => {
-			if (a.fields.includes('title') && !b.fields.includes('title')) return -1;
-			if (!a.fields.includes('title') && b.fields.includes('title')) return +1;
-			if (a.weight < b.weight) return +1;
-			if (a.weight > b.weight) return -1;
-			if (a.is_todo && a.todo_completed) return +1;
-			if (b.is_todo && b.todo_completed) return -1;
-			if (a.user_updated_time < b.user_updated_time) return +1;
-			if (a.user_updated_time > b.user_updated_time) return -1;
-			return 0;
-		});
+		// rows.sort((a, b) => {
```

Comment:
Is it not possible at this point to get the filters working while preserving the existing sorting algorithm? That would be ideal, as that way we can more easily separate and review the filter feature and the sorting feature.